packages/forms/src/Components/BaseFileUpload.php

```diff
@@ -234,6 +252,11 @@ public function getAcceptedFileTypes(): ?array
         return $types;
     }
 
+    public function getAllowReorder(): bool
```

Comment:
We can rename this to `canReorder()`, and the property can be renamed from `$this->allowReorder` to `$this->canReorder`